src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -176,7 +176,7 @@ public static function fromUrlEncodedString($serializedConfiguration)
         }
 
         foreach (['unsilenced', 'direct', 'indirect', 'self', 'other'] as $group) {
-            $verboseOutput[$group] = (bool) $normalizedConfiguration['verbose'];
+            $verboseOutput[$group] = filter_var($normalizedConfiguration['verbose'], FILTER_VALIDATE_BOOLEAN);
         }
 
         if (isset($normalizedConfiguration['quiet']) && \is_array($normalizedConfiguration['quiet'])) {
```

Comment:
looks like the docblock description misses telling about quiet btw :)

Reply:
I don't see what is missing, are you talking about `DeprecationErrorHandler` docblock or `fromUrlEncodedString` docblock please ?

Reply:
on top of DeprecationErrorHandler::register()

Reply:
Oh sorry ! I didn't pay attention to the `quiet` key. Actually I didn't know that option existed :)

Reply:
I added `quiet` to docblock.